grpc/src/test/java/com/linecorp/armeria/server/grpc/UnframedGrpcErrorHandlerTest.java

```diff
@@ -76,6 +78,60 @@ protected void configure(ServerBuilder sb) {
         }
     };
 
+    @RegisterExtension
+    static ServerExtension plainTextResServerWithBuilder = new ServerExtension() {
+        @Override
+        protected void configure(ServerBuilder sb) {
+            configureServer(sb, false,
+                            UnframedGrpcErrorHandler.builder()
+                                                    .build(),
+                            testService);
+        }
+    };
+
+    @RegisterExtension
+    static ServerExtension jsonResServerWithMarshalledMessage = new ServerExtension() {
+        @Override
+        protected void configure(ServerBuilder sb) {
+            configureServer(sb, false,
+                            UnframedGrpcErrorHandler.builder()
+                                                    .registerMarshalledMessageTypes(InternalError.class)
+                                                    .responseTypes(UnframedGrpcErrorResponseType.JSON)
+                                                    .build(),
+                            testServiceWithCustomMessage);
+        }
+    };
+
+    @RegisterExtension
+    static ServerExtension jsonResServerWithCustomJsonMarshaller = new ServerExtension() {
+        @Override
+        protected void configure(ServerBuilder sb) {
+            configureServer(sb, false,
+                            UnframedGrpcErrorHandler.builder()
+                                                    .jsonMarshaller(
+                                                            MessageMarshaller.builder()
+                                                                             .register(
+                                                                                     InternalError.class)
+                                                                             .build())
+                                                    .responseTypes(UnframedGrpcErrorResponseType.JSON)
+                                                    .build(),
+                            testServiceWithCustomMessage);
+        }
+    };
+
+    @RegisterExtension
+    static ServerExtension plainTextResServerWithCustomStatusMapping = new ServerExtension() {
```

Comment:
not plain text but plaintext
```suggestion
    static ServerExtension plaintextResServerWithCustomStatusMapping = new ServerExtension() {
```